visualforce-page1/question 5/StudentDetail.vfp

```diff
@@ -0,0 +1,2 @@
+<apex:page standardController="Student__c" extensions="StudentPDFController" action="{!attachPDF}">
```

Comment:
This question is not complete .
Below steps are missing
Create VF Page containing Student detail information (some general details)
On Click of this Link PDF will generated and attached with related Student and user can see any time that PDF. 
IF PDF already there then delete old one and create new one.

Reply:
Done.